controllers/api/book-routes.js

```diff
@@ -0,0 +1,16 @@
+const router = require('express').Router();
+
+router.get("/books", (req, res) => {
+    let results = books;
+    if (req.query) {
+      results = filterByQuery(req.query, results);
+    }
+    res.json(results);
+  });
```

Comment:
These are placeholders until the models are ready. Will update for books and movies at that point.